ProjectTraning/Task1/Driver.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProjectTraning.Task1
+{
+    class Driver: Worker
+    {
+
+        private double driverExperience;
+
+        private DriverLicenceCategories driverLicence;
+
+        public Driver(double driverExperience, DriverLicenceCategories driverLicence, string name, int age, decimal salary)
+
+            : base(name, age, salary)
+
+        {
+            this.driverExperience = driverExperience;
+
+            this.driverLicence = driverLicence;
+
+            Name = name;
```

Comment:
Missed 'this.' for all instance methods and properties.
Seems this was forgotten also.